logic/auth.go

```diff
@@ -39,6 +39,30 @@ func HasSuperAdmin() (bool, error) {
 	return false, err
 }
 
+// GetUsersDB - gets users
+func GetUsersDB() ([]models.User, error) {
```

Comment:
is this `func GetUsers() ([]models.ReturnUser, error) {` but with a different user type? could have a single function and an adapter from `models.User` to `models.ReturnUser` so we don't need to maintain duplicate code